packages/server-admin-ui/package.json

```diff
@@ -61,5 +61,12 @@
     "ci": "prettier --check src/",
     "clean": "rimraf ./public",
     "bundle-analyzer": "webpack-bundle-analyzer --port 4200 public/stats.json"
+  },
+  "dependencies": {
```

Comment:
These should be devDepencies?

Reply:
Please fix this.

Reply:
I am not sure I understand. These are dependencies for the UI and the new Appstore. What do you mean by "please fix this" ? These npm packages are dependencies of the new app store

Reply:
Admin ui is published to npm in bundled format. The ui does not need these to be installed and used.

Reply:
Not sure I understand because the new app store is using Ag-grid and fortawesome. The new app store won't work without these one installed, that's why they are called dependencies. 
I have poor knowledge of the release management of signal k so maybe you will have a better idea on what to do with this package.json. It doesn't seem very straightforward to me

Reply:
>you will have a better idea on what to do with this package.json

Make them devDepencies.